packages/@react-aria/interactions/stories/useFocusRing.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import {Cell, Column, Row, TableBody, TableHeader, TableView} from '@react-spectrum/table';
+import React, {useState} from 'react';
+import {SearchField} from '@react-spectrum/searchfield';
+import {storiesOf} from '@storybook/react';
+
+
+let manyColumns = [];
+for (let i = 0; i < 100; i++) {
+  manyColumns.push(
+    i === 0
+      ? {name: 'Column name', key: 'C0'}
+      : {name: 'Column ' + i, key: 'C' + i}
+  );
+}
+
+let manyRows = [];
+for (let i = 0; i < 1000; i++) {
+  let row = {key: 'R' + i};
+  for (let j = 0; j < 100; j++) {
+    row['C' + j] = j === 0 ? `Row ${i}` : `${i}, ${j}`;
+  }
+
+  manyRows.push(row);
+}
+
+storiesOf('useFocusRing', module)
+.add(
+  'search + table',
+  () => (
+    <SearchExample />
+  ),
+  {chromatic: {disable: true}}
```

Comment:
you shouldn't need this? did you copy it from elsewhere? may have some cleanup to do on that